learning.md

```diff
@@ -0,0 +1,24 @@
+## Learning and knowledge
```

Comment:
I'd call this 'Internal workshops'

Reply:
so I rename the file as well?

Reply:
yup

Reply:
I think `Learning and Knowledge Sharing` is better suited, as internal workshops are only a way to accomplish the goal of learning and knowledge sharing, we may add other kind of initiatives in the future